### PR TITLE
43 tour faq fragment

### LINKED ISSUE #43: Tour - FAQ fragment
There is an FAQ section at the end of all pages in the Tour area. You can see it for instance in this url: https://www.realmadrid.com/en/tour-bernabeu

The FAQ section will be implemented as a fragment document and included in all pages.

Screenshot 2023-05-05 at 17 41 22

Screenshot 2023-05-05 at 17 41 22

### PR DESCRIPTION
**Pull request by kacemait**
Hi team, I've pushed my latest changes to the '43-tour-faq-fragment' branch. This includes an implementation of the 'faq fragment' using a basic HTML/CSS code for the horizontally scrolling tabs on mobile. An alternative is to use an external library like Swiper.js, which provides these advanced features out of the box. However, including an external library could potentially impact the performance of our site, and we'd need to carefully consider this trade-off.

I wanted to raise this issue for discussion. If we believe the enhanced user experience outweighs the potential performance cost, I can proceed with integrating the external library. Otherwise, we can stick with the current CSS-based solution and look to enhance it down the line.

Looking forward to hearing your thoughts on this matter.

Fix #43 #Tour - FAQ fragment 

Test URLs:
- Before: https://main--realmadrid--hlxsites.hlx.page/tour/es/fragments/faq
- After: https://43-tour-faq-fragment--realmadrid--hlxsites.hlx.page/tour/es/fragments/faq

### CODE REVIEW

**Review by chicharr — COMMENTED** [review]
@kacemait overall looks good, just few comments

**Comment by vdua** [comment]
I am not sure whether we should call this column. I prefer it should have been a separate block in itself where the authoring would be similar. A tabs block would be good as well, but I don't feel it is a columns block.

Also I am not sure we are supporting swipe right now and making all the categories visible

image

Regarding swiper.js, I think it is a very cool library but provides a ton of functionality that is not needed right now. LHS will complain about the unused code.

**Comment by kacemait** [comment]
I have just pushed the requested changes.

**Comment by chicharr** [comment]
> Also I am not sure we are supporting swipe right now 
@msagolj  is working on an alternative solution to swipe

File: styles/styles.css [hunk 1]
```diff
@@ -59,7 +59,7 @@
 
 
   /** heading color */
-  --heading-color: #333;
+  --text-color: #333;
```
Comment (chicharr) [inline styles/styles.css:62]: this might be a misunderstanding. 
`--heading-color`  and `--text-color` are both needed and have a different value

**Review by chicharr — COMMENTED** [review]
@kait Added a couple of comments more,. Please address them, fix conflicts, and we are good to merge

**Pull request merged**